In [7]:
import pandas as pd
import csv

#reads archive using read_csv function
df = pd.read_csv('titanic.csv')

#displays first lines using head function
print(df.head())

#prints from lines 0 to 15 the columns Name, Age, Sex, and Survived
print(df.loc[0:15,['Name', 'Age', 'Sex', 'Survived']])
#prints from the first 5 lines the columns Survived, Pclass and Age
print(df[['Survived', 'Pclass', 'Age']].head())
#prints from lines 15 to 25 the columns Pclass and Fare
print(df.loc[15:25,['Pclass', 'Fare']])
#prints from lines 20 to 25 the columns Pclass and Embarked
print(df.loc[20:25], ['Pclass', 'Embarked'])
print("describe")
print(df.describe())




   PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
  

En el describe, se genera un resumen estadístico de las columnas, la cantidad de datos (count), el promedio (mean), la desviación estándar (std), el valor mínimo (min), los percentiles, así como el valor máximo (max), hay 891 registros en total, pero solo 714 tienen edad. la edad promedio de los pasajeros es de 29.7 años, con una desviación estándar de 14.5, un mínimo de 0.42 y un máximo de 80, mientras que la mediana es de 28. En cuanto a la supervivencia, alrededor del 38% de los pasajeros sobrevivieron. La mayoría viajaba en tercera clase, pues la mediana de la variable *Pclass* es 3. 